src/lib/output/themes/default/partials/member.declaration.tsx

```diff
@@ -29,12 +29,44 @@ export function memberDeclaration(context: DefaultThemeRenderContext, props: Dec
 
     const visitor = { reflection: renderTypeDeclaration };
 
+    /** Fix for #2717. If type is the same as value the type is omited */
+    function shouldRenderType() {
+        if (props.type && props.type.type === "literal") {
+            const typeObject = props.type.toObject();
+            const value = typeObject.value;
+            if (!value) {
+                // should be unreachable
+                return true;
+            }
+            if (typeof value === "object") {
```

Comment:
I don't think this is the logic we want. This is the case for bigint literals. TypeDoc splits them up as they can't be JSON serialized, but in this case we'd want to check that it is different.